packages/desktop-client/src/components/filters/FiltersMenu.js

```diff
@@ -264,15 +266,43 @@ function ConfigureField({
             />
           )}
 
-          <Stack direction="row" justify="flex-end" align="center">
-            <View style={{ flex: 1 }} />
+          <Stack
+            direction="row"
+            justify="flex-end"
+            align="center"
+            style={{ marginTop: 15 }}
+          >
+            {field === 'account' ||
+            field === 'payee' ||
+            field === 'notes' ||
+            field === 'category' ? (
+              <View
+                style={{
+                  flex: 1,
+                  flexDirection: 'row',
+                  alignItems: 'center',
+                  userSelect: 'none',
+                }}
+              >
+                <Checkbox
+                  id="add_negate"
+                  checked={selectNegate}
+                  onChange={() => {
+                    setSelectNegate(!selectNegate);
+                  }}
+                />
+                <label htmlFor="add_negate">Not</label>
```

Comment:
Added it here because there was already space without increasing the size of the menu. Didn't think about it being covered since the apply button was also covered.